### PR TITLE
 Set touchmove event not to be passive in order to allow preventDefault 

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/859
[Problem] When dragging AppBar there is an error on console:
"Unable to preventDefault inside passive event"
[Remark] Since Chrome 56 [1] all events are passive by default.
You can not call preventDefault on passive event.
Toggling DropDownMenu calls preventDefault.
[Solution] Modify touchmove event as not passive one.

[1] https://developers.google.com/web/updates/2017/01/scrolling-intervention

### CODE REVIEW

**Comment by HunseopJeong** [comment]
[291f880d](https://github.com/Samsung/TAU/pull/860/commits/291f880d1ba4ca6914adab15a870958993fbd2b8) doesn't seems to be a PR to solve the eslint issue only. Is it right?

File: src/js/core/event/gesture/Manager.js [hunk 1]
```diff
@@ -39,79 +39,79 @@
 			 * @private
```
Comment (TomaszLukawskiSam) [inline src/js/core/event/gesture/Manager.js]: Wrong indents in comments block

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

**Comment by TomaszLukawskiSam** [comment]
> [291f880d](https://github.com/Samsung/TAU/pull/860/commits/291f880d1ba4ca6914adab15a870958993fbd2b8) doesn't seems to be a PR to solve the eslint issue only. Is it right?

Yes. You can check this also in PR options:
image
Mostly white space

File: src/js/core/event/gesture/Manager.js [hunk 2]
```diff
@@ -140,7 +140,7 @@
 					var self = this;
 
 					if (self.touchSupport) {
-						document.removeEventListener("touchmove", self, false);
+						document.removeEventListener("touchmove", self, {passive: false}, false);
```
Comment (TomaszLukawskiSam) [inline src/js/core/event/gesture/Manager.js]: remove "useCapture"

**Comment by HunseopJeong** [comment]
> > [291f880d](https://github.com/Samsung/TAU/pull/860/commits/291f880d1ba4ca6914adab15a870958993fbd2b8) doesn't seems to be a PR to solve the eslint issue only. Is it right?
> 
> Yes. You can check this also in PR options:
> image
> Mostly white space

Yes. You are right.
Split view is better than unified :D
Thank you

File: src/js/core/event/gesture/Manager.js (hunk 2 above)
Comment (lslachciak) [reply src/js/core/event/gesture/Manager.js]: done

**Comment by lslachciak** [comment]
as discussed with @TomaszLukawskiSam I fixed code and default to nonpassive for all touch events

**Review by HunseopJeong — APPROVED** [review]
LGTM but you need to check @TomaszLukawskiSam comments.

File: src/js/core/event/gesture/Manager.js (hunk 1 above)
Comment (lslachciak) [reply src/js/core/event/gesture/Manager.js]: done

**Comment by lslachciak** [comment]
Fixed also indent problems for comments as @TomaszLukawskiSam pointed

**Review by TomaszLukawskiSam — APPROVED** [review]

**Comment by TomaszLukawskiSam** [comment]
Looks good to me.

**Pull request merged**